frontend/components/commons/forms/questions-form/fields/multi-label/MultiLabel.component.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class="wrapper">
+    <div class="title-area --body2">
+      <span v-text="title" v-optional-field="isRequired ? false : true" />
+
+      <BaseIconWithBadge
+        class="icon-info"
+        v-if="!!tooltipMessage"
+        icon="info"
+        :id="`${title}Rating`"
+        :show-badge="false"
+        iconColor="#acacac"
+        badge-vertical-position="top"
+        badge-horizontal-position="right"
+        badge-border-color="white"
+        v-tooltip="{ content: tooltipMessage, backgroundColor: '#FFF' }"
+      />
+    </div>
+
+    <MultiLabelSelectionComponent
+      :options="uniqueOptions"
+      :componentId="questionId"
+      :showSearch="showSearch"
+      :multiple="true"
+      :maxOptionsToShowBeforeCollapse="maxOptionsToShowBeforeCollapse"
+      @change="$emit('update:options', options)"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  name: "MultiLabelComponent",
+  props: {
+    questionId: {
+      type: String,
+      required: true,
+    },
+    title: {
+      type: String,
+      required: true,
+    },
+    options: {
+      type: Array,
+      required: true,
+    },
+    isRequired: {
+      type: Boolean,
+      default: () => false,
+    },
+    tooltipMessage: {
+      type: String,
+      default: () => "",
+    },
+    visibleOptions: {
+      type: Number | null,
+      required: false,
+    },
+  },
+  model: {
+    prop: "options",
+  },
+  data() {
+    return {
+      uniqueOptions: [],
+    };
+  },
+  beforeMount() {
+    this.uniqueOptions = this.options.reduce((accumulator, current) => {
+      if (!accumulator.find((item) => item.id === current.id)) {
+        accumulator.push(current);
+      }
+      return accumulator;
+    }, []);
+  },
+  computed: {
+    showSearch() {
+      return this.uniqueOptions.length >= 12;
```

Comment:
Create a constant instead of a magic number. I saw this number also for SingleLabel... You should use the constant in both places
```suggestion
      return this.uniqueOptions.length >= constans.OPTIONS_THRESHOLD_TO_ENABLE_SEARCH;
```